microsoft/SqlNexus
# Best practices Report rules addition and report improvements. Plus query performance improvements for post-import

- Added sp_configure rules to call out certain config values
- Multiple improvements in Best Practices report and back-end table rules (visual changes, tablix changes, colors, report parameters)
- Performance optimizations on the Perfstats queries (various) to speed up that phase

To test:
1. Examine the Best Practices report in SQL Nexus after you have imported a collection and see if you have any feedback
2. To check for the performance optimizations improvement look at the SQL Nexus log and check how long the time is between the start and end of Perfstats analysis

![image](https://user-images.githubusercontent.com/23519517/213356965-090eda71-4113-4343-afa7-0ed971b6e693.png)

For example in my case the time difference was about 2 seconds. Used to be about 3 min before this change.

SQLNexus Information: 0 : Running Perfstats Analysis
    DateTime=2023-01-19T0**4:30:16.1623753**ZSQLNexus Information: 0 : End of Perfstats Analysis
    DateTime=2023-01-19T**04:30:18.1270947**ZYou can see it took about 2 seconds in my case
